fmriprep/run_workflow.py

```diff
@@ -138,7 +134,7 @@ def main():
         subject_list = [op.basename(subdir)[4:] for subdir in glob.glob(
             op.join(settings['bids_root'], 'sub-*'))]
 
-    logger.info("subject list: {}", ', '.join(subject_list))
+    logger.info("subject list: " + str(subject_list))
```

Comment:
logging should be done with ```logger.info('some message with format %s tokens %s', string1, string2)```.

You are right, the former version didn't work because of the curly braces, so please replace this with ```logger.info("subject list: %s", ', '.join(subject_list))```

Finally, I would avoid the explicit use of ```str``` whenever possible, since it is different in python 2 and 3